### DISCUSSION TITLE
Fancontrol keeps crashing everyday

### DISCUSSION THREAD

**Discussion by infowege** [Rem0o/FanControl.Releases#2157] [General]
Quelle
FanControl

Zusammenfassung
Nicht mehr funktionsfähig

Datum
‎26.‎11.‎2023 14:31

Problemsignatur
Problemereignisame:	CLR20r3
Problemsignatur 01:	FanControl.exe
Problemsignatur 02:	174.0.0.0
Problemsignatur 03:	c63f3d0e
Problemsignatur 04:	FanControl
Problemsignatur 05:	174.0.0.0
Problemsignatur 06:	c63f3d0e
Problemsignatur 07:	14
Problemsignatur 08:	4c
Problemsignatur 09:	System.NullReferenceException
Betriebsystemversion:	10.0.22631.2.0.0.256.48
Gebietsschema-ID:	1031
Zusatzinformation 1:	e35e
Zusatzinformation 2:	e35e827e6acfd1d25039555aaf63a14a
Zusatzinformation 3:	6bfb
Zusatzinformation 4:	6bfb5909d1c43c95eec601cd16c576ed

![image](https://github.com/Rem0o/FanControl.Releases/assets/50275529/1651ebaf-32d4-41e4-94dc-9afdae4eff4f)

**Comment by infowege** [comment]
Issues: Stop working at startup, Stop working while playing,
My steps to resolve:
Autostart link not working
Aorus pro b550 v.1 motherboard with smartfan 5 disabled
Exe in the download Folder changed to c:
Programm version 174
Hardware:
RTX 3080 TI, Ryzen 3700x, 32Gb ram.
Open error log An unexpected error has been added to the logfile.

**Comment by Rem0o** [comment]
https://github.com/Rem0o/FanControl.Releases/issues/2158
